scripts/jenkins-yoctobuild-build.sh

```diff
@@ -384,9 +387,11 @@ fi
 # -----------------------
 
 if [ -d $WORKSPACE/meta-mender/meta-mender-commercial ]; then
-    RECIPE=$(ls $WORKSPACE/meta-mender/meta-mender-commercial/recipes-mender/mender-binary-delta | sort | tail -n1)
+    RECIPE=$(ls $WORKSPACE/meta-mender/meta-mender-commercial/recipes-mender/mender-binary-delta/*.bb | sort | tail -n1)
     mkdir -p $WORKSPACE/mender-binary-delta
-    s3cmd get --recursive s3://$(sed -e 's,delta_,delta/,; s/\.bb$//' <<<$RECIPE)/ $WORKSPACE/mender-binary-delta/
+    s3cmd get --recursive s3://$(sed -e 's,.*/,,; s,delta_,delta/,; s/\.bb$//' <<<$RECIPE)/ $WORKSPACE/mender-binary-delta/
+    chmod ugo+x $WORKSPACE/mender-binary-delta/x86_64/mender-binary-delta
+    chmod ugo+x $WORKSPACE/mender-binary-delta/x86_64/mender-binary-delta-generator
```

Comment:
Nitpick: this could have been squashed in the previous commit, as it doesn't belong go the recipe selection.

Reply:
I disagree. This change is unrelated to the previous commit, but is needed in this one because we introduce the call to the binary to find the version.

Reply:
@kacf Oh, that is right. It makes sense that follows "your journey".